### PR TITLE
Added my courses to free-courses-zh

### PR DESCRIPTION
**Pull request by chickenCHRISpy**
## What does this PR do?
Add resource(s)

### Description
Added Computer Network and OSI model in free-courses-zh file.

### How do we know it's really free?
All resources are from youtube with high rating.

### For book lists, is it a book? For course lists, is it a course? etc.
A one video tutorial.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-zh.md [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
 
 
 ### 0 - MOOC
 
 * [freeCodeCamp](https://chinese.freecodecamp.org)
 
 
+### Computer Networking
+
+* [Kunal Kushwaha](https://www.youtube.com/watch?v=IPvYjXCsTg8&ab_channel=KunalKushwaha) - Kunal Kushwaha
```
Comment (shrn01) [inline courses/free-courses-zh.md:16]:
```suggestion
* [Kunal Kushwaha](https://www.youtube.com/watch?v=IPvYjXCsTg8) - Kunal Kushwaha
```

File: courses/free-courses-zh.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+
+
 ### Linux
 
 * [Linux 核心設計](https://youtube.com/playlist?list=PL6S9AqLQkFpongEA75M15_BlQBC9rTdd8) - jserv
+
+
+### OSI Model
+
+* [Hussein Nasser](https://www.youtube.com/watch?v=eNF9z5JNl-A&ab_channel=HusseinNasser) - Hussein Nasser
```
Comment (shrn01) [inline courses/free-courses-zh.md:26]:
```suggestion
* [Hussein Nasser](https://www.youtube.com/watch?v=eNF9z5JNl-A) - Hussein Nasser
```

File: courses/free-courses-zh.md [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
+* [NetworkChuck](https://www.youtube.com/watch?v=oIRkXulqJA4&feature=youtu.be&ab_channel=NetworkChuck) - NetworkChuck
```
Comment (shrn01) [inline courses/free-courses-zh.md:27]:
```suggestion
* [NetworkChuck](https://www.youtube.com/watch?v=oIRkXulqJA4) - NetworkChuck
```

**Review by shrn01 — CHANGES_REQUESTED** [review]
This file is for chinese language. Please move the courses to the proper file

Also please provide the Names which are provided in the resources

**Comment by chickenCHRISpy** [comment]
> 

Sorry but I was asked to put my courses in this file itself previously. A issue was raised with the suggested changes.

**Comment by eshellman** [comment]
Please read docs/CONTRIBUTING.md guidelines!

**Pull request closed without merging**